internal/resolver/dummy.go

```diff
@@ -58,11 +57,6 @@ func (r *dummyResolver) Pusher(ctx context.Context, ref string) (remotes.Pusher,
 }
 
 // Discoverer returns a new discoverer for the provided reference
```

Comment:
Do you want to remove thsi?

Reply:
Wait this entire file? It's used for dry--run?

Reply:
Ohh okay github rendered this weird. It's not used